scipy/linalg/flapack.pyf.src

```diff
@@ -1188,6 +1188,98 @@ interface
 
    end subroutine <prefix2c>gesvd_lwork
 
+   subroutine <prefix2>gels(trans,m,n,nrhs,a,lda,b,ldb,work,lwork,info)
+
+   ! lqr,x,info = gels(a,b,lwork=..,overwrite_a=False,overwrite_b=False)
+   ! Solve Minimize 2-norm(A * X - B).
+
+     fortranname <prefix2>gels
+     callstatement (*f2py_func)(trans,&m,&n,&nrhs,a,&lda,b,&ldb,work,&lwork,&info)
+     callprotoargument char*,int*,int*,int*,<ctype2>*,int*,<ctype2>*,int*,<ctype2>*,int*,int*
+
+     character optional,intent(in),check(*trans=='N'||*trans=='T'):: trans = 'N'
+     integer depend(b),intent(hide):: nrhs = shape(b,1)
+     integer depend(a),intent(hide):: lda = shape(a,0)
+     integer intent(hide),depend(a):: m = shape(a,0)
+     integer intent(hide),depend(a):: n = shape(a,1)
+     integer depend(b),intent(hide):: ldb = shape(b,0)
+
+     <ftype2> dimension(m,nrhs),intent(in,out,copy,out=x) :: b
+     <ftype2> dimension(m,n),intent(in,out,copy,out=lqr) :: a
+     <ftype2> depend(lwork),dimension(MAX(lwork,1)),intent(hide,cache):: work
+     integer intent(out)::info
```

Comment:
There's no declaration for `lwork` here?
Maybe it works without, but I'd be explicit here.